solr/solr-ref-guide/modules/query-guide/pages/common-query-parameters.adoc

```diff
@@ -284,6 +284,17 @@ The query above allows you to examine the scoring explain info of the top matchi
 
 The default value of this parameter is blank, which causes no extra "explain info" to be returned.
 
+== partialResults Parameter
+
+This parameter controls Solr's behavior when a query execution limit is reached (e.g. `timeAllowed` or `cpuAllowed`).
+
+When this parameter is set to `true` (default) then even though reaching a limit terminates further query processing
+Solr will still attempt to return partial results collected so far. These results may be incomplete in
```

Comment:
In our ascii-doc files, generally start new sentences on new lines.  Easier to read diffs in the future.

Also this sentence is missing an "a" before "non-deterministic"

Reply:
Polish has no determiners ... Polish has no need for determiners :)